.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
@daffl I think that is good. At least for now. If no one has taken it right away here.